frontend/src/modules/_shared/parameterUtils.ts

```diff
@@ -0,0 +1,18 @@
+import type { ContinuousParameter } from "@framework/EnsembleParameters";
+import { ParameterType } from "@framework/EnsembleParameters";
+import type { RegularEnsemble } from "@framework/RegularEnsemble";
+
+export function getContinuousParameterArray(regularEnsemble: RegularEnsemble): ContinuousParameter[] | null {
+    const parameterArr = regularEnsemble.getParameters().getParameterArr();
+    const parameters: ContinuousParameter[] = [];
+    if (parameterArr) {
+        parameterArr.forEach((parameter) => {
+            if (parameter.isConstant || parameter.type !== ParameterType.CONTINUOUS) {
```

Comment:
As it's a shared util. Consider adding prop `excludeConstant = true` and make  exclude constant parameters optional (default true) to clarify it's also a filtering criteria. If not added to prop, consider clarifying by adjusting the function name